spec/services/akid_parser_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe AkidParser do
+  let(:valid_mailing_id) {'14203'}
+  let(:valid_actionkit_user_id) {'7145943'}
+  let(:valid_akid) {"#{valid_mailing_id}.#{valid_actionkit_user_id}.Si3iNO"}
+  let(:valid_return) { {mailing_id: valid_mailing_id, actionkit_user_id: valid_actionkit_user_id}}
+  let(:invalid_return) { {mailing_id: nil, actionkit_user_id: nil} }
+
+  it 'correctly parses a valid akid' do
+    expect(AkidParser.new(valid_akid).invalid?).to eq false
```

Comment:
For me it's a readability thing, and that's why matchers exist - to make specs read more naturally.
Also, we've been using the `be` matcher, so consistency might be important too.

However, it's such a minor point, I'd be perfectly happy if you ignored my comments on this entirely.